docs/Policy/Documentation/Scenario_Documentation_Policy.md

```diff
@@ -62,53 +71,65 @@ Please feel free to add any additional information or links that may be helpful
 ```
 
 #### Purpose
+
 This section of the documentation should outline the purpose of this scenario. While it may seem obvious to us, OpenShift CI is used by many teams and they may stumble across this scenario. It is important that people outside of CSPI and PQE know what this configuration does and why it is being run.
 
 This section can be as short as a sentence or two:
 
-<sub><sup>`ci-operator/config/windup/windup_integration_test/README.md`</sup></sub>
+`ci-operator/config/windup/windup_integration_test/README.md`
+
 ```markdown
 ## Purpose
 To provision the necessary infrastructure and use that infrastructure to execute MTR interop tests on pre-release OCP builds. The results of theses tests will be reported to the appropriate sources following execution.
 ```
 
 #### Process
+
 This section of your documentation should describe how this scenario works. The bulk of the scenario's documentation will be done here. Be sure to provide a brief overview of the steps this scenario takes to run and why they are necessary.
 
 ##### Cluster Provisioning and Deprovisioning<!-- omit from toc -->
+
 Under the "Process" header of your documentation be sure to include how and where the test cluster for this scenario is being provisioned and deprovisioned. 
 
 This could be as simple as linking to a workflow being utilized to do this work as the workflow (and it's documentation) may not be maintained by CSPI or PQE:
```

Comment:
```suggestion
This could be as simple as linking to a workflow being utilized to do this work as the workflow (and its documentation) may not be maintained by CSPI or PQE:
```

Reply:
Resolved in [**39fe241**7dca0aa2df5e3e0cb4676f4fa7ac99bf7](https://github.com/CSPI-QE/ocp-ci-docs/pull/10/commits/39fe2417dca0aa2df5e3e0cb4676f4fa7ac99bf7)